afteraction-share.html

```diff
@@ -200,6 +200,38 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 		</div>
 	</div>
 	{% endif %}
+
+	{% if page.custom_fields.share_whatsapp_message %}
+		<script>
+		var whatsappMsg = '{{ page.custom_fields.share_whatsapp_message }} ';
+		</script>
+	{% else %}
+		<script>
+		var whatsappMsg = '{{ page.followup.share_title_value }} ';
+		</script>
+	{% endif %}
+
+	<script>
+		jQuery(document).ready(function($){
+			$('.whatsapp-share').on('click', function(e) {
+				e.preventDefault();
+				// Set Whatsapp Share URL based on response from API
+				var getShareURL = '/share/link?type=ot&action_id={{action.id}}&short=1&page_name={{page.name}}';
```

Comment:
noticing that both action.id and page.name are provided here, in contrast with the other "manual" copy link implementation which only sends action.id. Is there a particular reason for this small diff?

Reply:
The differences in the links are based on this table: https://www.notion.so/How-to-create-trackable-share-links-e82f7e785ebb4133bb67ac94925d0567#961ea90f69b44700b2f776c87535b4d8

Reply:
ah ok, i thought this whatsapp share was on a thank you page like the other manual copy link implementation, but perhaps actually it's technically this category?:

> On an ActionKit page (sharing another page, replacing shortname with the page shortname)

But if that's accurate, the table shows akid being sent, not action_id

Reply:
hmm you are right that it should be the "sharing the page that's just been signed" category, but I think it should be fine as is. I remember testing this out and having issues when only `action_id` and not `page_name` was included.